README.md

```diff
@@ -6,117 +6,78 @@
 [![Build Sphinx](https://github.com/IntelPython/dpnp/workflows/Build%20Sphinx/badge.svg)](https://intelpython.github.io/dpnp)
 [![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/IntelPython/dpnp/badge)](https://securityscorecards.dev/viewer/?uri=github.com/IntelPython/dpnp)
 
+<img align="left" src="https://spec.oneapi.io/oneapi-logo-white-scaled.jpg" alt="oneAPI logo" width="75"/>
+
 # DPNP - Data Parallel Extension for NumPy*
+
+Data Parallel Extension for NumPy* or `dpnp` is a Python library that
+implements a subset of NumPy* that can be executed on any data parallel device.
+The subset is a drop-in replacement of core NumPy* functions and numerical data types.
+
 [API coverage summary](https://intelpython.github.io/dpnp/reference/comparison.html#summary)
 
 [Full documentation](https://intelpython.github.io/dpnp/)
 
-[DPNP C++ backend documentation](https://intelpython.github.io/dpnp/backend_doc/)
+`Dpnp` is the core part of a larger family of [data-parallel Python libraries and tools](https://www.intel.com/content/www/us/en/developer/tools/oneapi/distribution-for-python.html)
+to program on XPUs.
 
-## Build from source:
-Ensure you have the following prerequisite packages installed:
 
-- `cython`
-- `cmake >=3.21`
-- `dpcpp_linux-64` or `dpcpp_win-64` (depending on your OS)
-- `dpctl`
-- `mkl-devel-dpcpp`
-- `onedpl-devel`
-- `ninja`
-- `numpy >=1.19,<1.25a0`
-- `python`
-- `scikit-build`
-- `setuptools`
-- `sysroot_linux-64 >=2.28` (only on Linux OS)
-- `tbb-devel`
+# Installing
 
-After these steps, `dpnp` can be built in debug mode as follows:
+You can install the library using `conda` or [pip](https://pypi.org/project/dpnp/)
+package managers. It is also available as part of the [Intel(R) Distribution for Python](https://www.intel.com/content/www/us/en/developer/tools/oneapi/distribution-for-python.html)
+(IDP).
 
-```bash
-git clone https://github.com/IntelPython/dpnp
-cd dpnp
-python scripts/build_locally.py
-```
+## Intel(R) oneAPI
 
-## Install Wheel Package via pip
-Install DPNP
-```cmd
-python -m pip install --index-url https://software.repos.intel.com/python/pypi dpnp
-```
+You can find the most recent release of `dpnp` every quarter as part of the Intel(R) oneAPI releases.
 
-Set path to Performance Libraries in case of using venv or system Python:
-```cmd
-export LD_LIBRARY_PATH=<path_to_your_env>/lib
-```
+To get the library from the latest oneAPI release, follow the
+instructions from Intel(R) [oneAPI installation guide](https://www.intel.com/content/www/us/en/developer/articles/guide/installation-guide-for-oneapi-toolkits.html).
 
-It is also required to set following environment variables:
-```cmd
-export OCL_ICD_FILENAMES_RESET=1
-export OCL_ICD_FILENAMES=libintelocl.so
-```
+> **NOTE:** You need to install the Intel(R) oneAPI AI Analytics Toolkit to get
```

Comment:
Has its official name changed to AI Tools?
https://www.intel.com/content/www/us/en/docs/oneapi-ai-analytics-toolkit/get-started-guide-linux/2024-2/overview.html

Reply:
Replaced with a reference towards IDP installation guide.